spec/factories/openscap_result.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :openscap_result_skip_callback, :class => OpenscapResult do
+    # This callback can be very annoying and handicaping in tests
+    after(:build) { |r| r.class.skip_callback(:save, :before, :create_rule_results) }
```

Comment:
Can you go into more detail?  Without an example, it's hard to understand why we would skip callbacks.

Reply:
This call back parses the binary blob. This requires the binary blob to be of a real Scap result file, and creates unwanted rule_results out of it. In most tests it would be much easier to create them manually instead.
This is limiting because the well structured SCAP file is not short, and we don't necessarily want the extra rules or to have this file in the repo.
What do you think that we will create here two factories?  one normal without disabling the callback?